deployment/tf/modules/yc/logs/configs/output-elasticsearch.conf

```diff
@@ -2,7 +2,8 @@
     Name            yc-logging
     Match           kube*
     resource_id     {kubernetes/container_name}
-    message_key     message
+    message_key     log
+    stream_name     {stream}
```

Comment:
The main idea is to split logs to different streams - app, stdout and stderr. After this, each stream can be enabled or disabled in YC Logging UI.